_docs/faqs.md

```diff
@@ -29,6 +30,10 @@ Either way works.
 Yes! The DAC now accepts any science variables that have a valid [CF Standard Name](http://cfconventions.org/standard-names.html). Any ancillary variables (as specifed in the variable attribute `ancillary_variables`) will also be ingested into ERDDAP.
 
 
+## What is the current procedure for sending data to the Global Telecommunications System (GTS)?
+
+The National Data Buoy Center harvests new profile observations from the DAC's ERDDAP server once per hour, encodes the profiles in to a modified drifting buoy BUFR format and releases the messages to the GTS.  The development of a glider specific BUFR format is currently being developed and finalized.
```

Comment:
can you add a hyper link to the DAC ERDDAP page on the mention of ``DAC's ERDDAP server```

Reply:
working on it.